docs/en/query_language/functions/other_functions.md

```diff
@@ -861,12 +861,69 @@ Takes the states of the aggregate function and returns a column with values, are
 For example, takes state of aggregate function (example runningAccumulate(uniqState(UserID))), and for each row of block, return result of aggregate function on merge of states of all previous rows and current row.
 So, result of function depends on partition of data to blocks and on order of data in block.
 
-## joinGet('join_storage_table_name', 'get_column', join_key) {#other_functions-joinget}
+## joinGet {#joinget}
 
-Gets data from [Join](../../operations/table_engines/join.md) tables using the specified join key.
+The function lets you extract data from the table the same way as from a [dictionary](../dicts/index.md).
+
+Gets data from [Join](../../operations/table_engines/join.md#creating-a-table) tables using the specified join key.
 
 Only supports tables created with the `ENGINE = Join(ANY, LEFT, <join_keys>)` statement.
 
+**Syntax** 
+
+```sql
+joinGet(join_storage_table_name, `value_column`, join_keys)
+```
+
+**Parameters** 
+
+- `join_storage_table_name` — [Identifier](../../syntax/#syntax-identifiers) indicates where search is performed. In most cases, this is the name of the database and/or table. Firstly, identifier is searched in the default database (see parameter `default_database` in the config file), and then in other databases. To override the default database, use the `USE db_name` command.
```

Comment:
> Firstly, identifier is searched in the default database (see parameter `default_database` in the config file), and then in other databases.

It could lead to misunderstanding. Afaik we search only in default database. But the defult database could be set in different way. The text here says we search one by one in different places. If so it's probably a bug. If database is set we should use it. If it's not set we should use default database. And throw an error if default dtabase has no table with such name.

Reply:
Checked, everything as you wrote. Thanks!